pygeoapi/provider/xarray_edr.py

```diff
@@ -213,16 +219,65 @@ def cube(self, **kwargs):
                 data.coords[self.x_field].values[-1],
                 data.coords[self.y_field].values[-1]
             ],
-            "time": [
-                _to_datetime_string(data.coords[self.time_field].values[0]),
-                _to_datetime_string(data.coords[self.time_field].values[-1])
-            ],
+            "time": time,
             "driver": "xarray",
             "height": height,
             "width": width,
-            "time_steps": data.dims[self.time_field],
+            "time_steps": time_steps,
             "variables": {var_name: var.attrs
                           for var_name, var in data.variables.items()}
         }
 
         return self.gen_covjson(out_meta, data, self.fields)
+
+    def _make_datetime(self, datetime_):
+        """
+        Make xarray datetime query
+        :param datetime_: temporal (datestamp or extent)
+        :returns: xarray datetime query
+        """
+        datetime_ = datetime_.rstrip('Z').replace('Z/', '/')
+        if '/' in datetime_:
+            begin, end = datetime_.split('/')
+            if begin == '..':
+                begin = self._data[self.time_field].min().values
+            elif end == '..':
```

Comment:
Although it would be redundant to do `?datetime=../..`, this would throw an error. Changing `elif end == '..':` to `if end == '..':` would prevent.